src/dynamo.js

```diff
@@ -50,15 +50,16 @@ export class Dynamo {
     await client.batchWriteItem(params).catch((e) => console.error('Error while deleting stale users', e))
   }
 
-  async createGame(participants) {
+  async createGame(players) {
     const id = crypto.randomUUID()
     const params = {
       TableName: TABLE_GAMES,
       Item: marshall({
         id: id,
-        participants: participants,
-        playerTurn: participants[Math.round(Math.random())],
-        state: STARTED
+        players: players,
```

Comment:
Currently we are passing a string array to the players, which does not conform to the protocol. Here is the code we came up with in the code review:
```js
  async createGame(players) {
    const shuffledPlayers = players.sort(() => Math.random() - 0.5)

    const [first, second] = shuffledPlayers

    const playerOne = this.getPlayerById(first)
    playerOne.piece = X
    const playerTwo = this.getPlayerById(second)
    playerTwo.piece = O

    const params = {
      TableName: TABLE_GAMES,
      Item: marshall({
        id: crypto.randomUUID(),
        players: [playerOne, playerTwo],
        playerTurn: first,
        state: STARTED,
        board: [EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY, EMPTY]
      })
    }
    await client.putItem(params)
  }
```